app/src/main/res/values/strings.xml

```diff
@@ -1659,20 +1660,21 @@ Partially means that a wheelchair can enter and use the restroom, but no handrai
     <string name="bicycle_boulevard_is_a">It’s a %1$s</string>
     <string name="bicycle_boulevard_is_not_a">It’s not a %1$s</string>
     <string name="bicycle_boulevard">Bicycle boulevard</string>
-    <string name="separate_cycleway_no">No cycling</string>
-    <string name="separate_cycleway_allowed">Cyclists are allowed</string>
+    <!-- Focus on cycling being the thing that is explicitly disallowed -->
+    <string name="separate_cycleway_no_signed">A sign explicitly prohibits cycling</string>
+    <string name="separate_cycleway_footway_allowed_sign">Footway, but a sign allows cycling</string>
     <!-- highway=path / foot=yes + bicycle=yes -->
-    <string name="separate_cycleway_path">Path or trail</string>
+    <string name="separate_cycleway_path">Generic path or trail (Not designated for anyone in particular)</string>
     <!-- highway=footway / foot=designated and anything else that is not designated for cyclists-->
-    <string name="separate_cycleway_no_or_allowed">Not designated for cyclists (cycling may still be allowed)</string>
+    <string name="separate_cycleway_no_or_allowed">Footway, no explicit sign about cycling (whether it is allowed depends on legislation)</string>
     <!-- foot=designated + bicycle=designated -->
-    <string name="separate_cycleway_non_segregated">Shared-use path</string>
+    <string name="separate_cycleway_non_segregated">Designated shared-use path</string>
     <!-- foot=designated + bicycle=designated + segregated=yes -->
-    <string name="separate_cycleway_segregated">Separated bike and foot path</string>
+    <string name="separate_cycleway_segregated">Designated separated bike and foot path</string>
     <!-- highway=cycleway / bicycle=designated -->
-    <string name="separate_cycleway_exclusive">Bike-only path</string>
+    <string name="separate_cycleway_exclusive">Designated bike-only path</string>
     <!-- highway=cycleway + sidewalk=* -->
-    <string name="separate_cycleway_with_sidewalk">Bike-only path with adjacent sidewalk (separated by a curb)</string>
+    <string name="separate_cycleway_with_sidewalk">Designated bike-only path with adjacent sidewalk (separated by a curb)</string>
```

Comment:
Changed the strings

Reply:
Right, thanks, but I tent towards removing "Designated" from the last three options again.

For the shared-use path, it is important to have "designated" to distinguish it from the "path or trail".

For the others, I don't think it does add value as there is no risk of mix-ups with other values. To the contrary, I think it might actually **add** potential for confusion: Bike paths are _always_ designated for cyclists in one way or another, otherwise it would not be a bike path (but a path or trail), so no need to explicitly state it. However, using the word "designated" (=DE: _gekennzeichnet_) may invoke the notion in users that to choose this option, the path must be *signed* as a bike path, which is not required. E.g. many bike paths in Germany are not signed but are distinguishable from the footway by different (often red) paving - whether these bike paths are "_de: gekennzeichnet_" as such rather than just _recognizable_ as such is a topic people may butt their heads about. (After all, there indeed was a long discussion about it in the forum, whether `*=designated` means that it is signed in one way or another.)

Reply:
Removed the last three "Designated"s